pkgs/by-name/br/breakpointHook/attach.sh

```diff
@@ -35,12 +66,16 @@ debugShell="$(getVar debugShell)"
 pwd="$(readlink /proc/$pid/cwd)"
 
 # enter the namespace of the failed build
+# bash needs to be executed with --init-file /build/env-vars to include the bash native
+#   variables like ones declared via `declare -a`.
+# If another shell is chosen via `debugShell`, it will only have simple env vars avaialable.
 exec nsenter --mount --net --target "$pid" "$bashInteractive" -c "
```

Comment:
IPC was missing the other day. UTS/PID might be also fairly safe.

```suggestion
exec nsenter --mount --ipc --uts --pid  --net --target "$pid" "$bashInteractive" -c "
```

After that it's mainly user namespaces that are missing.

Reply:
applied, tested, works